packages/docs-page/components/version-select/index.tsx

```diff
@@ -11,14 +11,20 @@ import {
 interface Props {
   versions: { label: string; name: string }[]
   basePath: string
+  projectName?: string
```

Comment:
Fair point. I had opted for this approach as I didn't feel we needed the full flexibility, but maybe it's not super clear what the prop is actually doing on usage 💭

Do you have a preference?

Reply:
Sorry missed your comment!

> Just one concern about the projectName prop name and the usage with string interpolation.
> 
> Curious if you think the ${projectName} Version string interpolation is potentially limiting to us, in the event that say Design Team wants some copy that doesn't end with " Version"

I think if design wants something else it would be consistent across projects, in which case it _felt_ better to enforce it in the component than to have to update all usages / implementations.